pages/lists/[listId]/contributors.tsx

```diff
@@ -17,48 +18,38 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   } = await supabase.auth.getSession();
   const bearerToken = session ? session.access_token : "";
 
-  const { listId, limit: rawLimit, range } = ctx.params as FilterParams;
+  const { listId } = ctx.params as FilterParams;
 
-  const limit = rawLimit ?? "10"; // Can pull this from the querystring in the future
-  const [{ data, error: contributorListError }, { data: list, error }] = await Promise.all([
-    fetchApiData<PagedData<DBListContributor>>({
-      path: `lists/${listId}/contributors?limit=${limit}&range=${range ?? "30"}`,
-      bearerToken,
-      pathValidator: validateListPath,
-    }),
-    fetchApiData<DBList>({ path: `lists/${listId}`, bearerToken, pathValidator: validateListPath }),
-  ]);
+  const { data: list, error } = await fetchApiData<DBList>({
```

Comment:
I've removed server-side rendering contributor list data and only kept critical stuff to load the page, e.g. list info.

Something related to the initial data for the contributor list is what caused the issue with the card view. Still trying to work out why, but regardless it makes sense to only load this client-side going forward as mentioned in the PR description.